src/components/Image/Image.tsx

```diff
@@ -81,6 +82,9 @@ export interface ImageProps
   additionalImageStyles?: { [key: string]: any };
   /** Alternate text description of the image */
   alt?: string;
+  /** Optional value to control the aspect ratio of the card image; default
+   * value is `"original"` */
+  aspectRatio?: ImageRatios;
```

Comment:
Question just to clarify, devs should use 'ratio' then?

Reply:
No, devs should keep using `aspectRatio` for the `Image` component. The `ImageWrapper` component should use `ratio` but this component is only used internally, so devs wouldn't be able to access it anyway.